recipes/prettytable/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: prettytable
+  version: "7"
```

Comment:
Wait, I send you a mail :)
Everything is fine here :) Just submit your second PR with the irods-client.